src/components/banner.js

```diff
@@ -20,14 +20,22 @@ function Banner(props) {
 
   useEffect(() => {
     getSnippets();
-  }, [1]);
+  }, []);
 
   useEffect(() => {
```

Comment:
This effect is not needed. 
Instead below block of code can be put at line number 53 i.e inside axios success callback. 

`if (response.data.factoids.length > 1) {
      setSnippet(response.data.factoids[0]);
    }`

Reply:
Thanks, fixed!